src/singletons/YieldFarmingDataStoreV2.js

```diff
@@ -321,19 +320,31 @@ export default class YieldFarmingDataStoreV2 {
   getUserDataPUSH = async (provider) => {
     return new Promise(async (resolve, reject) => {
       if (this.state.account) {
-        // Rewards Claimed
         const pushCoreV2 = this.state.pushCoreV2;
 
         let {
-          epochRewards, 
-          currentEpochNumber,
-          userStaked,
-          potentialReward,
-          availableRewards
-        } = await getUserPushStakingInfo(provider,this.state.account,addresses.pushCoreV2);
-
-        //Calculating TotalStaked Amount
+          epochRewards = bn(0), //BN
+          currentEpochNumber = 0,//BN
+          userStaked = bn(0),//BN
+          potentialReward = bn(0),//BN
+          availableRewards = bn(0),//BN
+        } = {};
+
+        ({ epochRewards, currentEpochNumber } = await getUserPushStakingInfo(provider, this.state.account, addresses.pushCoreV2));
+
+        //these values changes if the account changes.
+        if (this.state.account !== '0x0000000000000000000000000000000000000001') {
```

Comment:
Same as above. Why this exact number?

If we need it then use the above mentioned approach.

Reply:
Fixed in new commit